R/build-footer.R

```diff
@@ -24,17 +24,16 @@ footnote_components <- function(pkg = ".", call = caller_env()) {
     default = default_roles(),
     call = call
   )
-  authors <- data_authors(pkg, roles = roles)$main %>%
-    purrr::map_chr("name") %>%
-    paste(collapse = ", ")
+  authors <- data_authors(pkg, roles = roles)$main
+  authors_str <- paste(purrr::map_chr(authors, "name"), collapse = ", ")
```

Comment:
```suggestion
  authors_str <- toString(purrr::map_chr(authors, "name"))
```

Reply:
maybe less readable? but with this the variable and function names match :grin:

Reply:
I don't use `toString()` enough to know what it does.

Reply:
Right so that wouldn't help with readability but for the record here it'd do the same thing as `paste(, collapse = ", ")`

``` r
x <- c("thing", "something", "anything")
paste(x, collapse = ", ")
#> [1] "thing, something, anything"
```

``` r
toString(x)
#> [1] "thing, something, anything"
```

<sup>Created on 2024-06-03 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>